src/core/api_product/nodo_pagamenti_api/decoupler/decoupler-activate-outbound.xml

```diff
@@ -94,5 +94,36 @@
   </choose>
   </when>
   </choose>
+    <choose>
+      <when condition="@( context.Variables.GetValueOrDefault<bool>("is_wfesp", true) )">
+        <set-variable name="outcomeWFESPCart" value="@{
+                  XElement response = context.Response.Body.As<XElement>(preserveContent: true);
+                  XElement body = response.Descendants(response.Name.Namespace + "Body").FirstOrDefault();
+                  XElement primitive = (XElement) body.FirstNode;
+                  string outcomeWfespCart = (string) primitive.Descendants("esitoComplessivoOperazione").FirstOrDefault();
 
+                  return outcomeWfespCart.Equals("OK");
+                  }" />
+        <choose>
+          <when condition="@( context.Variables.GetValueOrDefault<bool>("outcomeWFESPCart", true) )">
+            <set-variable name="fixedUrlWFESP" value="@{
+                return "https://wfesp.pagopa.gov.it/redirect/wpl05/get?idSession=" + context.Response.Headers.GetValueOrDefault("sessionId");
```

Comment:
put `https://wfesp.pagopa.gov.it/redirect/wpl05/get?idSession="` into a named valued